### PR TITLE
Implement https://jira.hl7.org/browse/FHIR-43718 & https://jira.hl7.org/browse/FHIR-43719

### PR DESCRIPTION
**Pull request by dbojicic**
**Adoption of Gender Harmony Gender Identity data element (https://jira.hl7.org/browse/FHIR-43718)**
1. Added Individual Gender Harmony extension to AU Base Patient, AU Base RelatedPerson and AU Base Practitioner
- [Gender Identity Response value set](https://www.healthterminologies.gov.au/integration/R4/fhir/ValueSet/gender-identity-response-1) was updated in NCTS
- invariant is applied to state that individual-genderIdentity value shall be a member of the Gender Identity Response value set if any of the codes within that value set can apply as per #815 
- updated implementation guidance in AU Base Patient, and added to AU Base RelatedPerson and AU Base Practitioner
2. Replaced [patient-gender-identity](https://build.fhir.org/ig/hl7au/au-fhir-base/SearchParameter-patient-gender-identity.html) SearchParameter with a new gender-identity and updated Search Parameters .md page
3. Updated patient example and added sex-and-gender RelatedPerson and Practitioner examples

**Adoption of Gender Harmony Pronouns data element (https://jira.hl7.org/browse/FHIR-43719)**
1. Added Individual Pronouns to AU Base Patient, AU Base RelatedPerson and AU Base Practitioner
- equivalent of extensible binding is applied via an invariant on the root element to require that Individual Pronouns value shall be a member of the AU Pronouns value set if any of the codes within that value set can apply as per #815 
- updated intro content
2. Removed the manual inclusion of Individual Pronouns extension
3. Removed inclusion of the FHIR ValueSet Personal Pronouns and stu-notes related to the inclusion of this extension 
4. Updated patient example and added sex-and-gender RelatedPerson and Practitioner examples

* Addressed comments provided in https://github.com/hl7au/au-fhir-base/pull/826.

_NOTE: [Australian Pronouns](https://www.healthterminologies.gov.au/integration/R4/fhir/ValueSet/australian-pronouns-1) ValueSet is now published in NCTS. [Gender Identity Response](https://www.healthterminologies.gov.au/integration/R4/fhir/ValueSet/gender-identity-response-1) ValueSet has been updated to include additional concepts as documented in FHIR-43718 proposal._

### CODE REVIEW

File: input/pagecontent/terminology.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ This page lists all ValueSets and CodeSystems defined as part of this specificat
 
 As well as these included definitions this specification typically references [National Clinical Terminology Service (NCTS)](https://www.healthterminologies.gov.au/integration/R4/fhir) terminology resources. 
 
-<p class="stu-note">We have introduced some terminology to support extensions around sex and gender. These extensions are currently under discussion with HL7 Australia and/or HL7 International. The terminology resources introduced are: Personal Pronouns value set and Recorded Sex Or Gender Type value set. These extensions, and the associated terminology, may be removed or changed prior to publication of the next version of this guide.</p>
+<p class="stu-note">We have introduced some terminology to support extensions around sex and gender. These extensions are currently under discussion with HL7 Australia and/or HL7 International. The terminology resources introduced are: Recorded Sex Or Gender Type value set. These extensions, and the associated terminology, may be removed or changed prior to publication of the next version of this guide.</p>
```
Comment (dt-r) [inline input/pagecontent/terminology.md:6]: Why is this note still here? The pre-adopted terminology has been removed?

File: input/pagecontent/search-parameters.md [hunk 2]
```diff
@@ -12,4 +12,12 @@ NOTE: The following search parameters **SHALL NOT** be interpreted on their own
 ### Patient
 
 - [indigenous-status](SearchParameter-indigenous-status.html)
-- [patient-gender-identity](SearchParameter-patient-gender-identity.html)
```
Comment (dt-r) [inline input/pagecontent/search-parameters.md:15]: Missing the other resources the new search parameter applies to incl Person and PractitionerRole.

File: input/pagecontent/changes.md [hunk 3]
```diff
@@ -39,17 +37,32 @@ To help implementers, only the more significant changes are listed here.
...
 </ul>
 </li>
 <li>Changed Patient.extension in <a href="StructureDefinition-au-patient.html">AU Base Patient</a>: 
-<ul><li>to add Individual Pronouns and Person Recorded Sex Or Gender</li>
+<ul><li>to add Individual Pronouns (<a href="https://jira.hl7.org/browse/FHIR-43719">FHIR-43719</a>) and Person Recorded Sex Or Gender</li>
 <li>to add Australian Veteran Status</li>
-<li>to remove the value slice with the binding to Gender Identity Response (extensible) from genderIdentity (<a href="https://github.com/hl7au/au-fhir-base/issues/815">au-fhir-base #815</a>)</li>
+<li>to remove genderIdentity extension and replace it with the Individual Gender Identity extension (<a href="https://jira.hl7.org/browse/FHIR-43718">FHIR-43718</a>)</li>
 </ul>
 <li>Changed context of extension <a href="StructureDefinition-indigenous-status.html">Australian Indigenous Status</a> to add Person and RelatedPerson.</li>
```
Comment (dt-r) [inline input/pagecontent/changes.md:48]: Restructure the change log to be a list by profile for the profile changes to make this more readable,

File: input/intro-notes/StructureDefinition-au-practitioner-intro.md [hunk 4]
```diff
@@ -5,3 +5,13 @@
 - A tertiary qualification or professional membership (non-Ahpra-sourced data) is represented by `Practitioner.qualification` 
   - If none of the codes from the preferred value set are suitable then at least text should be sent in `Practitioner.qualification.code`
 - See each Identifier profile page for guidance related to that identifier type.
+- This profile supports the practitioner's gender identity aligned to [Australian Bureau of Statics Standard for Sex, Gender, Variations of Sex Characteristics and Sexual Orientation Variables, 2020](https://www.abs.gov.au/statistics/standards/standard-sex-gender-variations-sex-characteristics-and-sexual-orientation-variables/latest-release#gender) using the [Individual Gender Identity extension](http://hl7.org/fhir/StructureDefinition/individual-genderIdentity) and [Gender Identity Response](https://healthterminologies.gov.au/fhir/ValueSet/gender-identity-response-1) value set.
```
Comment (dt-r) [inline input/intro-notes/StructureDefinition-au-practitioner-intro.md:8]: Remove the references to specific (and incorrect) examples.

File: input/intro-notes/StructureDefinition-au-relatedperson-intro.md [hunk 5]
```diff
@@ -2,3 +2,12 @@
 
 **Profile specific implementation guidance:**
 - See each Identifier profile page for guidance related to that identifier type.
+- This profile supports the related person's gender identity aligned to [Australian Bureau of Statics Standard for Sex, Gender, Variations of Sex Characteristics and Sexual Orientation Variables, 2020](https://www.abs.gov.au/statistics/standards/standard-sex-gender-variations-sex-characteristics-and-sexual-orientation-variables/latest-release#gender) using the [Individual Gender Identity extension](http://hl7.org/fhir/StructureDefinition/individual-genderIdentity) and [Gender Identity Response](https://healthterminologies.gov.au/fhir/ValueSet/gender-identity-response-1) value set.
```
Comment (dt-r) [inline input/intro-notes/StructureDefinition-au-relatedperson-intro.md:5]: Remove the references to specific (and incorrect) examples.

File: input/ig.xml [hunk 6]
```diff
@@ -1379,6 +1387,14 @@
 			<description value="Shows a typical related person example as the father of a patient"/>
 			<exampleCanonical value="http://hl7.org.au/fhir/StructureDefinition/au-relatedperson"/>
 		</resource>
+		<resource>
+			<reference>
+				<reference value="RelatedPerson/example-sex-and-gender"/>
+			</reference>
+			<name value="RelatedPerson - as family member"/>
```
Comment (dt-r) [inline input/ig.xml:1394]: The name does not state this is for sex and gender and it should.

File: input/ig.xml [hunk 7]
```diff
@@ -2122,5 +2138,9 @@
 			<code value="special-url"/>
 			<value value="http://terminology.hl7.org.au/ValueSet/coverage-type-extended"/>
 		</parameter>
+		<parameter>
```
Comment (dt-r) [inline input/ig.xml:2141]: Addition of a duplicate code fragment - needs to be removed.

File: input/examples/patient-example-sex-and-gender.xml [hunk 8]
```diff
@@ -1,20 +1,24 @@
 <Patient xmlns="http://hl7.org/fhir">
     <id value="example-sex-and-gender"/>
-    <!--   This purpose of this example is to demonstrate the R5 pre-adopted extensions for sex and gender:
-    - Individual Pronouns
+    <!--   This purpose of this example is to demonstrate the use of sex and gender elements:
+    - Gender Harmony Gender Identity (Individual Gender Identity extension)
+    - Gender Harmony Pronouns (Individual Pronouns extension)
+    - and demonstrate the R5 pre-adopted extensions for sex and gender:
```
Comment (dt-r) [inline input/examples/patient-example-sex-and-gender.xml:6]: last bullet point isn't adjusted for the PR change. Should say Gender Harmony Recorded Sex or Gender (Person Recorded Sex Or Gender extension)

File: input/examples/patient-example-sex-and-gender.xml [hunk 9]
```diff
@@ -26,15 +30,7 @@
                 </coding>
             </valueCodeableConcept>
         </extension>
-        <extension url="period">
-            <valuePeriod>
```
Comment (dt-r) [inline input/examples/patient-example-sex-and-gender.xml:30]: Why is this period removed? No reason for removal in the PR.

File: input/examples/relatedperson-example-sex-and-gender.xml [hunk 10]
```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (dt-r) [inline input/examples/relatedperson-example-sex-and-gender.xml:1]: One of the examples needs two instances of gender identity. The RelatedPerson may be a good choice and use the period & comment in both instances. @reubendaniels may be able to supply meaningful input into the examples if the Gender Harmony IG does not offer sufficient background.

File: input/pagecontent/profiles-and-extensions.md [hunk 11]
```diff
@@ -16,7 +16,7 @@ The following profiles and have been defined for this implementation guide.
...
+<p class="stu-note">We have introduced support for the Person Recorded Sex Or Gender extension. It is published in this guide to pre-adopt the material published in the R5 <a href="http://hl7.org/fhir/extensions/1.0.0">FHIR Extensions Pack</a>. This extension is an R4 (4.0.1) conversion of the canonical extension and may contain forwards incompatible data types when converted to R4 (see the extension page in this guide for information on any  material differences). Implementers are advised to take note that this content may be removed or changed and to take caution if using this extension.</p>
```
Comment (dt-r) [inline input/pagecontent/profiles-and-extensions.md:19]: A note still needs to be present but this content is no longer relevant. See https://jira.hl7.org/browse/FHIR-44514 the FHIR extension data types are changing.

Change the tone of this note to be something around advice that what we have adopted in AU Base is gonna be same as the FHIR extension - and point to https://jira.hl7.org/browse/FHIR-44514.

You should watch that issue to get an update when it's applied and see what they've done. At minimum we would update the note but may be able to remove in its entirety.

File: input/pagecontent/terminology.md (hunk 1 above)
Comment (dt-r) [inline input/pagecontent/terminology.md:6]: A note may need to be present depending on outcome of review comments but this content is no longer relevant. 

None of this terminology is under discussion by either groups. None of the extensions are changing.

Question is why is this in the AU Base IG and not just a reference to the FHIR standard? Its technical.

File: input/resources/au-patient.xml [hunk 12]
```diff
@@ -104,20 +111,23 @@
       <path value="Patient.extension" />
       <sliceName value="genderIdentity" />
       <short value="The patient's gender identity shall be a member of the Gender Identity Response value set if any of the codes within that value set can apply"/>
```
Comment (dt-r) [inline input/resources/au-patient.xml:113]: Why say "the patient's". Too long a text. We don't do that for other invariants. Just say Gender Identity.

Comment (dt-r) [inline input/resources/au-patient.xml:113]: Shorten the text. Replace "The patient's gender" with "Gender"

File: input/resources/au-patient.xml [hunk 13]
```diff
@@ -104,20 +111,23 @@
...
         <code value="Extension" />
-        <profile value="http://hl7.org/fhir/StructureDefinition/patient-genderIdentity" />
+        <profile value="http://hl7.org/fhir/StructureDefinition/individual-genderIdentity" />
       </type>
       <condition value="inv-pat-1"/>
     </element>
     <element id="Patient.extension:individualPronouns">
       <path value="Patient.extension"/>
       <sliceName value="individualPronouns"/>
+      <short value="The patient's pronouns shall be a member of the Australian Pronouns value set if any of the codes within that value set can apply"/>
```
Comment (dt-r) [inline input/resources/au-patient.xml:124]: Shorten the text. Replace "The patient's pronouns" with "Pronouns"

File: input/resources/au-practitioner.xml [hunk 14]
```diff
@@ -18,7 +18,53 @@
...
+      <type>
+        <code value="Extension" />
+        <profile value="http://hl7.org/fhir/StructureDefinition/individual-genderIdentity" />
+      </type>
+      <condition value="inv-pra-3"/>
+    </element> 
+    <element id="Practitioner.extension:individualPronouns">
+      <path value="Practitioner.extension"/>
+      <sliceName value="individualPronouns"/>
+      <short value="The pracititioner's pronouns shall be a member of the Australian Pronouns value set if any of the codes within that value set can apply"/>
```
Comment (dt-r) [inline input/resources/au-practitioner.xml:60]: Shorten the text:

1. Replace "The practitioner's pronouns" with "Pronouns".
2. Replace "any of the" with "any codes"

File: input/resources/au-relatedperson.xml [hunk 15]
```diff
@@ -18,6 +18,52 @@
...
+      <type>
+        <code value="Extension" />
+        <profile value="http://hl7.org/fhir/StructureDefinition/individual-genderIdentity" />
+      </type>
+      <condition value="inv-relper-0"/>
+    </element> 
+    <element id="RelatedPerson.extension:individualPronouns">
+      <path value="RelatedPerson.extension"/>
+      <sliceName value="individualPronouns"/>
+      <short value="The related person's pronouns shall be a member of the Australian Pronouns value set if any of the codes within that value set can apply"/>
```
Comment (dt-r) [inline input/resources/au-relatedperson.xml:60]: Shorten the text:

1. Replace "The related person's pronouns" with "Pronouns".
2. Replace "any of the" with "any codes"

File: input/resources/au-relatedperson.xml [hunk 16]
```diff
@@ -18,6 +18,52 @@
...
+          <type value="value" />
+          <path value="url" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="RelatedPerson.extension:genderIdentity">
+      <path value="RelatedPerson.extension" />
+      <sliceName value="genderIdentity" />
+      <short value="The related person's gender identity shall be a member of the Gender Identity Response value set if any of the codes within that value set can apply"/>
```
Comment (dt-r) [inline input/resources/au-relatedperson.xml:49]: Shorten the text.

1. Replace "The related person's gender" with "Gender"
2. Replace "any of the" with "any codes"

**Review by dt-r — CHANGES_REQUESTED** [review]
Provided minor comments to address.

Resolving the below comment - there will be a PR on RSG very soon that will address this comment.

> In addition I do not have clarity on why individual pronouns pre-adoption was remove and not recorded sex or gender. Please reach out and provide comms on this difference from the JIRA and what is in the PR. Ensure that Brett Esler and Reuben Daniels are aware. Also update the PR with the rationale and difference.

> Will there be a later PR coming soon or is a technical difference to what the FHIR WG voted on required?

File: input/ig.xml (hunk 7 above)
Comment (dbojicic) [reply input/ig.xml:2141]: Removed duplication.

File: input/ig.xml [hunk 17]
```diff
@@ -1058,6 +1058,14 @@
 			<description value="Shows a typical Ahpra registered professional"/>
 			<exampleCanonical value="http://hl7.org.au/fhir/StructureDefinition/au-practitioner"/>
 		</resource>
+		<resource>
+			<reference>
+				<reference value="Practitioner/example-sex-and-gender"/>
+			</reference>
+			<name value="Practitioner - a practitioner with an identifier only"/>
```
[the remark this answers is not in the document]
Comment (dbojicic) [reply input/ig.xml:1065]: Updated name to state the example is for sex and gender

File: input/ig.xml (hunk 6 above)
Comment (dbojicic) [reply input/ig.xml:1394]: Updated name to state the example is for sex and gender

File: input/examples/patient-example-sex-and-gender.xml (hunk 8 above)
Comment (dbojicic) [reply input/examples/patient-example-sex-and-gender.xml:6]: Updated comment

File: input/examples/patient-example-sex-and-gender.xml (hunk 9 above)
Comment (dbojicic) [reply input/examples/patient-example-sex-and-gender.xml:30]: Reinstated accidentally removed code

File: input/examples/relatedperson-example-sex-and-gender.xml (hunk 10 above)
Comment (dbojicic) [reply input/examples/relatedperson-example-sex-and-gender.xml:1]: Preliminary change done, and running it by @reubendaniels

File: input/resources/au-patient.xml (hunk 12 above)
Comment (dbojicic) [reply input/resources/au-patient.xml:113]: Done

File: input/resources/au-patient.xml (hunk 13 above)
Comment (dbojicic) [reply input/resources/au-patient.xml:124]: Done

Comment (dbojicic) [reply input/resources/au-patient.xml:124]: Done

File: input/resources/au-practitioner.xml (hunk 14 above)
Comment (dbojicic) [reply input/resources/au-practitioner.xml:60]: Done

File: input/resources/au-practitioner.xml [hunk 18]
```diff
@@ -18,7 +18,53 @@
...
+          <type value="value" />
+          <path value="url" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="Practitioner.extension:genderIdentity">
+      <path value="Practitioner.extension" />
+      <sliceName value="genderIdentity" />
+      <short value="The practitioner's gender identity shall be a member of the Gender Identity Response value set if any of the codes within that value set can apply"/>
```
[the remark this answers is not in the document]
Comment (dbojicic) [reply input/resources/au-practitioner.xml:49]: Done

File: input/resources/au-relatedperson.xml (hunk 15 above)
Comment (dbojicic) [reply input/resources/au-relatedperson.xml:60]: Done

File: input/resources/au-relatedperson.xml (hunk 16 above)
Comment (dbojicic) [reply input/resources/au-relatedperson.xml:49]: Done

File: input/intro-notes/StructureDefinition-au-relatedperson-intro.md (hunk 5 above)
Comment (dbojicic) [reply input/intro-notes/StructureDefinition-au-relatedperson-intro.md:5]: Done

File: input/intro-notes/StructureDefinition-au-practitioner-intro.md (hunk 4 above)
Comment (dbojicic) [reply input/intro-notes/StructureDefinition-au-practitioner-intro.md:8]: Done

File: input/pagecontent/search-parameters.md (hunk 2 above)
Comment (dbojicic) [reply input/pagecontent/search-parameters.md:15]: Added Person and PractitionerRole to the list

File: input/pagecontent/profiles-and-extensions.md (hunk 11 above)
Comment (dt-r) [reply input/pagecontent/profiles-and-extensions.md:19]: Remove the following text:
> We have introduced support for the Person Recorded Sex Or Gender extension. It is published in this guide to pre-adopt the material published in the R5 <a href="http://hl7.org/fhir/extensions/1.0.0">FHIR Extensions Pack</a>.

Also remove
> Implementers are advised to take note that this content may be removed or changed and to take caution if using this extension

Replace:
> This extension is an R4 (4.0.1) conversion of the canonical extension and may contain forwards incompatible data types when converted to R4 (see the extension page in this guide for information on any  material differences). 

with

> The <a href="StructureDefinition-individual-recordedSexOrGender.html">Person Recorded Sex Or Gender</a> is temporarily included in AU Base as an R4 (4.0.1) conversion of the canonical extension published in the R5 <a href="http://hl7.org/fhir/extensions/1.0.0">FHIR Extensions Pack</a>. When available in the R4 extension pack this inclusion will be removed and AU Base will point to the extension pack. No impact to vendors is expected.

File: input/ig.xml [hunk 19]
```diff
@@ -1058,6 +1058,14 @@
 			<description value="Shows a typical Ahpra registered professional"/>
 			<exampleCanonical value="http://hl7.org.au/fhir/StructureDefinition/au-practitioner"/>
 		</resource>
+		<resource>
+			<reference>
+				<reference value="Practitioner/example-sex-and-gender"/>
+			</reference>
+			<name value="Practitioner - a practitioner with an identifier only, for use in sex and gender"/>
```
Comment (dt-r) [inline input/ig.xml:1065]: remove 'with an identifier only,'

File: input/intro-notes/StructureDefinition-au-practitioner-intro.md [hunk 20]
```diff
@@ -5,3 +5,5 @@
 - A tertiary qualification or professional membership (non-Ahpra-sourced data) is represented by `Practitioner.qualification` 
   - If none of the codes from the preferred value set are suitable then at least text should be sent in `Practitioner.qualification.code`
 - See each Identifier profile page for guidance related to that identifier type.
+- When exchanging concepts of sex or gender, refer to the guidance in the [Gender Harmony Implementation Guide](http://hl7.org/xprod/ig/uv/gender-harmony/).
```
Comment (dt-r) [inline input/intro-notes/StructureDefinition-au-practitioner-intro.md:8]: Missing the implementation guidance that was present. Put back but change it to remove the reference to the examples.

File: input/intro-notes/StructureDefinition-au-relatedperson-intro.md [hunk 21]
```diff
@@ -2,3 +2,5 @@
 
 **Profile specific implementation guidance:**
 - See each Identifier profile page for guidance related to that identifier type.
+- When exchanging concepts of sex or gender, refer to the guidance in the [Gender Harmony Implementation Guide](http://hl7.org/xprod/ig/uv/gender-harmony/).
```
Comment (dt-r) [inline input/intro-notes/StructureDefinition-au-relatedperson-intro.md:5]: Missing the implementation guidance that was present. Put back but change it to remove the reference to the examples.

File: input/resources/au-patient.xml [hunk 22]
```diff
@@ -103,21 +110,24 @@
...
-        <profile value="http://hl7.org/fhir/StructureDefinition/patient-genderIdentity" />
+        <profile value="http://hl7.org/fhir/StructureDefinition/individual-genderIdentity" />
       </type>
       <condition value="inv-pat-1"/>
     </element>
     <element id="Patient.extension:individualPronouns">
       <path value="Patient.extension"/>
       <sliceName value="individualPronouns"/>
+      <short value="Pronouns shall be a member of the Australian Pronouns value set if any codes within that value set can apply"/>
+      <comment value="Individual Pronouns [Australian Pronouns](http://terminology.hl7.org.au/ValueSet/au-pronouns) ([Extensible](http://hl7.org/fhir/R4/terminologies.html#extensible))."/>
```
Comment (dt-r) [inline input/resources/au-patient.xml:125]: wrong url for the value set - see qa.html for all the instances where this issue is present.

File: input/examples/patient-example-sex-and-gender.xml [hunk 23]
```diff
@@ -25,7 +29,6 @@
                     <display value="she/her/her/hers/herself"/>
                 </coding>
             </valueCodeableConcept>
-        </extension>
```
Comment (dt-r) [inline input/examples/patient-example-sex-and-gender.xml:28]: There is a missing tag in this example I think it is this one - see qa.html

File: input/pagecontent/changes.md (hunk 3 above)
Comment (dt-r) [reply input/pagecontent/changes.md:48]: This has not been addressed. Should be:
- Change to AU Base Patient:
   - change
   - change
- Changes to AU Base Practitioner:
   - change
   - change that is a group of changes:
      - change
      - change

Comment (dbojicic) [reply input/pagecontent/changes.md:48]: Done

File: input/pagecontent/terminology.md (hunk 1 above)
Comment (dt-r) [reply input/pagecontent/terminology.md:6]: I will resolve this. An imminent PR on RSG will address this.

File: input/pagecontent/changes.md (hunk 3 above)
Comment (dt-r) [reply input/pagecontent/changes.md:48]: Might not be pushed. The file in this PR does not have this content.

File: input/intro-notes/StructureDefinition-au-practitioner-intro.md (hunk 20 above)
Comment (dbojicic) [reply input/intro-notes/StructureDefinition-au-practitioner-intro.md]: Reinstated guidance & removed links to examples.

File: input/pagecontent/changes.md (hunk 3 above)
Comment (dt-r) [reply input/pagecontent/changes.md:48]: I'll double check

File: input/resources/au-patient.xml (hunk 13 above)
Comment (dt-r) [reply input/resources/au-patient.xml]: 1. Confirmed fix
2. Not yet fixed

File: input/resources/au-practitioner.xml (hunk 14 above)
Comment (dt-r) [reply input/resources/au-practitioner.xml]: 1. Confirmed fix
2. Not yet fixed

File: input/intro-notes/StructureDefinition-au-relatedperson-intro.md (hunk 21 above)
Comment (dt-r) [reply input/intro-notes/StructureDefinition-au-relatedperson-intro.md]: I see it reinstated but have lost the following point that needs to be present:
- When exchanging concepts of sex or gender, refer to the guidance in the [Gender Harmony Implementation Guide](http://hl7.org/xprod/ig/uv/gender-harmony/).

File: input/intro-notes/StructureDefinition-au-practitioner-intro.md (hunk 20 above)
Comment (dt-r) [reply input/intro-notes/StructureDefinition-au-practitioner-intro.md]: I see it reinstated but have lost the following point that needs to be present:
- When exchanging concepts of sex or gender, refer to the guidance in the [Gender Harmony Implementation Guide](http://hl7.org/xprod/ig/uv/gender-harmony/).

Comment (dbojicic) [reply input/intro-notes/StructureDefinition-au-practitioner-intro.md]: Reinstated lost point.

File: input/resources/au-patient.xml (hunk 22 above)
Comment (dbojicic) [reply input/resources/au-patient.xml:125]: This has been fixed, will push shortly.

File: input/examples/patient-example-sex-and-gender.xml (hunk 23 above)
Comment (dbojicic) [reply input/examples/patient-example-sex-and-gender.xml:28]: This has been addressed, will push shortly

File: input/ig.xml (hunk 19 above)
Comment (dbojicic) [reply input/ig.xml:1065]: Removed.

File: input/examples/patient-example-sex-and-gender.xml (hunk 8 above)
Comment (dbojicic) [reply input/examples/patient-example-sex-and-gender.xml:6]: Removed

**Comment by dbojicic** [comment]
@dt-r, please re-review. Changes for AU Base adoption of RSG will be provided in a separate PR.

File: input/pagecontent/changes.md (hunk 3 above)
Comment (dt-r) [reply input/pagecontent/changes.md]: We can merge and then fix after.

**Pull request merged**